l10n_es_extra_data/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,2 @@
+Módulo que nos permite configurar impuestos que no se nos permite añadir a
+en esta versión de odoo de la localización española.
```

Comment:
Quizá la idea no es únicamente impuestos, ¿no? Podrían ser cuentas, posiciones fiscales, o cambios en datos ya existentes.

¿Qué opináis?

Reply:
- Sí, esto sería para cualquier cosa del plan contable que salga nuevo, por ejemplo #1744

Reply:
Mirad a ver si os gusta ahora la descripción. @HaraldPanten @pedrobaeza